tests/operator/operator.go

```diff
@@ -111,9 +111,12 @@ type vmYamlDefinition struct {
 	vmSnapshots   []vmSnapshotDef
 }
 
+var (
+	originalKv  *v1.KubeVirt
+	originalCDI *cdiv1.CDI
+)
```

Comment:
I'm not sure that exposing this as a package-level variable is better, TBH.
Your reasoning is:

>There is no reason to use lambdas here, because there is no risk for naming pollution, as there are no other files in this package.

TBH I'm not sure this is a good reasoning. What if we'll have another test there in the future?

If you want to remove the lambda function here, I you can go for something like that and keep the variable scoped as before:
```go
func copyOriginalKv(originalKv *v1.KubeVirt) *v1.KubeVirt {
```